### ISSUE TITLE
IncludeSender in Codeunit 1380 - Batch Processing Mgt.

### ISSUE THREAD

**Issue by hemisphera** [microsoft/ALAppExtensions#3373]
The events that `Codeunit 1380 Batch Processing Mgt.` provides are quite useless. None of them have `IncludeSender` -  which basically eliminates the possibility to check **who** exactly is batch processing **what** and **how**.

Take `OnVerifyRecord` as an example. Suppose I am running the report `Batch Post Sales Orders`. All that this event gives me is the Sales Header being processed. But I have absolutely no idea or way to find out what Codeunit and what parameters exactly are being used. With this event only I am unable to check if it was really `Batch Post Sales Orders` or maybe even `Batch Post Sales Invoices`. Was Invoice set? Or only Ship? ReplacePostingDate? All the parameters are there, inside the codeunit. They are just not accessible to a subscriber.

tl;dr:
Add `IncludeSender` to all events on `Codeunit 1380 Batch Processing Mgt.`.
